x-pack/plugins/reporting/server/core.ts

```diff
@@ -350,6 +338,36 @@ export class ReportingCore {
     return startDeps.esClient;
   }
 
+  public getScreenshots(options: ScreenshotOptions): Rx.Observable<ScreenshotResult> {
+    return Rx.defer(() => this.getPluginStartDeps()).pipe(
+      pluck('screenshotting'),
+      switchMap((screenshotting) => {
+        const config = this.getConfig();
+        return screenshotting.getScreenshots({
+          ...options,
+
+          timeouts: {
+            loadDelay: durationToNumber(config.get('capture', 'loadDelay')),
+            openUrl: durationToNumber(config.get('capture', 'timeouts', 'openUrl')),
+            waitForElements: durationToNumber(config.get('capture', 'timeouts', 'waitForElements')),
+            renderComplete: durationToNumber(config.get('capture', 'timeouts', 'renderComplete')),
+          },
+
+          layout: {
+            zoom: config.get('capture', 'zoom'),
+            ...options.layout,
```

Comment:
It feels like all of these config options could be moved over and given the `renameFromRoot` treatment, as the others have. Let's do that in another PR, unless you object.